src/utils/constants.ts

```diff
@@ -0,0 +1,16 @@
+export const CONSTANTS = {
+  GLOBAL_PREFIX: "api",
+};
+
+export const SWAGGER_CONSTANTS = {
+  TITLE: "API",
+  DESCRIPTION: "WPCAS Survey Service API description",
```

Comment:
Change the description according to service

Reply:
Done